services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -206,10 +212,12 @@ const CMSUserRoutes = ({
                         path={RoutesRecord.SUBMISSIONS_SUMMARY}
                         element={<SubmissionSummary />}
                     />
-                    <Route
-                        path={RoutesRecord.SUBMISSIONS_MCCRSID}
-                        element={<MccrsId />}
-                    />
+                    {showMCCRSRecordNumber && (
+                        <Route
+                            path={RoutesRecord.SUBMISSIONS_MCCRSID}
+                            element={<MccrsId />}
+                        />
+                    )}
```

Comment:
This change caused a flicker of the 404 not found page before going to the MCCRSID page.

https://github.com/Enterprise-CMCS/managed-care-review/assets/98117700/4d149859-bf88-4ece-962c-7f9c5b3af696